src/util/dateTime.ts

```diff
@@ -0,0 +1,17 @@
+export const getDateTime = (date: string, offset = 0) => {
```

Comment:
could consider using `moment` if we need more date stuff in the future

Reply:
Yep, this works and required no peer deps which is nice - something to consider though